spring-cloud-task-integration-tests/pom.xml

```diff
@@ -36,5 +36,14 @@
 			<artifactId>spring-cloud-task-batch</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.batch</groupId>
+			<artifactId>spring-batch-core</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-deployer-resource-support</artifactId>
+		</dependency>
```

Comment:
We may want to exclude integrations tests  from the main set of modules.  And create a maven profile for running integration tests.